ext/travisci/install_puppetserver.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+lein=lein2
+
+# Update a single dependency in a leiningen project.clj file.
+update_dependency_var() {
+    local file="$1"
+    local varname="$2"
+    local new_version="$3"
+
+    SED_ADDRESS="(def $varname"
+    SED_REGEX="\".*\""
+    SED_REPLACEMENT="\"$new_version\""
+    SED_COMMAND="s|$SED_REGEX|$SED_REPLACEMENT|"
+
+    set -e
+    set -x
+
+    sed -i -e "/$SED_ADDRESS/ $SED_COMMAND" $file
+}
+
+lein-pprint() {
+    export PUPPETSERVER_HEAP_SIZE=1G
```

Comment:
OK, well there's still likely something wrong here because aside from the fact that having it on the same line should work, the variable has already been exported by the time this code runs (see line 31) -- it's very suspicious if it fails when the setting in this function is removed.